spec/rails_helper.rb

```diff
@@ -5,6 +5,7 @@
 require 'rspec/rails'
 require 'shoulda/matchers'
 require 'cancan/matchers'
+include DeviseMasquerade::Controllers::UrlHelpers
```

Comment:
`Rails.application.routes.url_helpers` ?  it doesn't work le, and it was included by default ?